### ISSUE TITLE
Game crashes on boot on Nintendont

### ISSUE THREAD

**Issue by NElectron** [etaylor8086/prime-randomizer-web#1]
Version: 0.2.0

Whenever I try to load the randomized rom using Nintendont on a softmodded Wii, the game will start booting and show the Nintendo logo for a split second before crashing to a green screen. I checked to see if maybe I was using a bad rom or something like that, but I tried the original rom that I use for the randomizer and it works fine. I also tried using the randomizer a couple times, but each generated rom will have the same exact effect when used on Nintendont. The game works fine when used with Dolphin though; I'm only having issues when using Nintendont.

I'm just wondering is there anything I need to change in the Nintendont settings to get it to work, or is it something going wrong between the loader and the rom?

EDIT: using an older version of Nintendont seems to fix the problem, so this seems like an issue with Nintendont(?)

**Comment by etaylor8086** [comment]
Hi, thanks for reporting this.

I ran into this issue myself not too long ago and investigated earlier this week. Starting with Nintendont v5.477, games patched with the randomprime patcher will hit this green screen crash, so it is recommended to roll back to or add [Nintendont v5.476](https://github.com/FIX94/Nintendont/tree/89b64cb7f3a9d774d23826afdf90a4e623102cbd) to your SD card's `apps` folder.

See [this randomprime issue I opened](https://github.com/aprilwade/randomprime/issues/5) for more details.
